avplayback.js

```diff
@@ -116,20 +116,20 @@ instance.prototype.actions = function(system) {
 			]
 		},
 
-		'play':     { label: 'Play standby clip' },
-		'pause':    { label: 'Pause Resume' },
-		'stop':     { label: 'Stop' },
-		'previewStart':     { label: 'Play preview clip' },
-		'previewPause':    { label: 'Preview Pause' },
-		'previewStop':     { label: 'Preview Stop' },
-		'freeze':   { label: 'Freeze temp' },
-		'loop':     { label: 'Loop temp' },
-		'nextTag':   { label: 'Next Tag' },
-		'prevTag':     { label: 'Previous Tag' },
-		'10':       { label: 'Goto 10' },
-		'20':       { label: 'Goto 20' },
-		'30':       { label: 'Goto 30' },
-		'60':       { label: 'Goto 60' }
+		'play':     		{ label: 'Play standby clip' },
+		'pause':    		{ label: 'Pause Resume' },
+		'stop':     		{ label: 'Stop' },
+		'previewStart':     	{ label: 'Play preview clip' },
+		'previewPause':    	{ label: 'Preview Pause' },
+		'previewStop':     	{ label: 'Preview Stop' },
+		'freeze':   		{ label: 'Freeze temp' },
+		'loop':     		{ label: 'Loop temp' },
+		'nextTag':   		{ label: 'Next Tag' },
+		'prevTag':     		{ label: 'Previous Tag' },
+		'10':       		{ label: 'Goto 10' },
+		'20':       		{ label: 'Goto 20' },
+		'30':       		{ label: 'Goto 30' },
+		'60':       		{ label: 'Goto 60' }
 	});
```

Comment:
A tip for nex time: On start of lines, we use tabs. But inside lines, we use spaces. But I am unsure if it is in our coding guidelines. Anyways. No biggie :)

Reply:
Okay cool. I know editors themselves may have caused the issues. I jumped around editors a bit as well as different machines. I will try to be more consistent with some of these next changes and PRs I'm planning to make with some of these other updates I plan to get moved in. 

I originally made the changes using vim on one of our production machines, a few more changes in sublime text, and then cleaned up using githubs editor.

Thanks for being reasonable and having a great system you got started here!

Reply:
No problem, thank you for contributing!